api/BluetoothRemoteGATTService.json

```diff
@@ -125,27 +121,9 @@
             "ie": {
               "version_added": false
             },
-            "opera": [
```

Comment:
I agree with simplifying all of this repeated OS and flag information since it's already on the parent feature. But it does mean that information won't be visible on a page like https://developer.mozilla.org/en-US/docs/Web/API/BluetoothRemoteGATTService/getCharacteristic#browser_compatibility.

@ddbeck how do you think we should deal with cases like this?

Reply:
There's some disagreement on this point. See https://github.com/mdn/browser-compat-data/issues/6509 for the broader issue.

My approach to flags has been to only represent the flag on the feature that requires the flag, not its descendants (unless the flags differ). For example, a CSS property behind a flag doesn't need to duplicate the flag data on subfeatures for property values. This does lead to a poorer presentation for individual interface member pages, however. In those cases I don't think it's too bad: for `SomeFlaggedInterface.method()`, you'd get an informative `SomeFlaggedInterface is not defined` error.

But since there's no consensus, I'm happy to allow anything that looks reasonable. If you think it'd be preferable to have flags inherit, then go for it, but I'm not going to ask for it myself.

Reply:
I'll go along with removing the flags on subfeatures here. I don't know what I think should be the guideline :)